pkg/route/api/v1/types.go

```diff
@@ -41,7 +41,11 @@ type RouteSpec struct {
 
 	// To is an object the route points to. Only the Service kind is allowed, and it will
 	// be defaulted to Service.
-	To kapi.ObjectReference `json:"to"`
+	To RouteTargetReference `json:"to"`
+
+	// AdditionalTos is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	AdditionalTos []RouteTargetReference `json:"additionalTos,omitempty"`
```

Comment:
Talked to jordan a bit.

Since we're going to switch to ingresses eventually, we should probably pick terminology that is consistent with ingresses.  Ingress uses `Backend` to describe the service and port to target.  HAProxy also has backends, and that roughly is analogous to what the field is adding.  Since we have a primary (`to`) we should distinguish that this field doesn't contain the primary, so my suggestion at this point would be 

    AlternateBackends []RouteTargetReference `json:"alternateBackends,omitempty"`